scipy/optimize/slsqp.py

```diff
@@ -159,19 +159,29 @@ def fmin_slsqp(func, x0, eqcons=(), f_eqcons=None, ieqcons=(), f_ieqcons=None,
 
     Notes
     -----
-    Exit modes are defined as follows ::
-
-        -1 : Gradient evaluation required (g & a)
-         0 : Optimization terminated successfully
-         1 : Function evaluation required (f & c)
-         2 : More equality constraints than independent variables
-         3 : More than 3*n iterations in LSQ subproblem
-         4 : Inequality constraints incompatible
-         5 : Singular matrix E in LSQ subproblem
-         6 : Singular matrix C in LSQ subproblem
-         7 : Rank-deficient equality constraint subproblem HFTI
-         8 : Positive directional derivative for linesearch
-         9 : Iteration limit reached
+    Exit modes are defined as follows:
+
+    -1 : Gradient evaluation required (g & a)
+
+    0 : Optimization terminated successfully
+
+    1 : Function evaluation required (f & c)
+
+    2 : More equality constraints than independent variables
+
+    3 : More than 3*n iterations in LSQ subproblem
+
+    4 : Inequality constraints incompatible
+
+    5 : Singular matrix E in LSQ subproblem
+
+    6 : Singular matrix C in LSQ subproblem
+
+    7 : Rank-deficient equality constraint subproblem HFTI
+
+    8 : Positive directional derivative for linesearch
+
+    9 : Iteration limit reached
```

Comment:
Elsewhere we use `*` in lists like this (e.g. see several cases being edited in this PR).  That allows use to avoid the blanks lines.  Since we're already using that style in several places, we might as well use it here, too.
```suggestion
    * -1 : Gradient evaluation required (g & a)
    * 0 : Optimization terminated successfully
    * 1 : Function evaluation required (f & c)
    * 2 : More equality constraints than independent variables
    * 3 : More than 3*n iterations in LSQ subproblem
    * 4 : Inequality constraints incompatible
    * 5 : Singular matrix E in LSQ subproblem
    * 6 : Singular matrix C in LSQ subproblem
    * 7 : Rank-deficient equality constraint subproblem HFTI
    * 8 : Positive directional derivative for linesearch
    * 9 : Iteration limit reached
```
(I think this change is in scope for this PR, because it is part of fixing the markup of lists.)

Reply:
... but I'll let y'all decide if that is worthwhile--this is not a blocker for me.  Even as it is now, this PR is a big improvement!

Reply:
> Elsewhere we use * in lists like this (e.g. see several cases being edited in this PR).

Yes, in some, but definitely not all. The only place where there is any consistency, as far as I've seen is within `linprog`, where bullets are _not_ used.

Since there is not yet consensus about which style is best, I [wrote above](https://github.com/scipy/scipy/pull/13106#discussion_r528389459) that I'd decided not to add or remove bullets elsewhere, so I'd like to leave this one alone.